README.md

```diff
@@ -192,3 +192,7 @@ If threads are getting reused it will clear the current mdc of thread before cop
 ### Contributing
 Refer [contribution guidline](CONTRIBUTING.md)
 
+### Blogs
+- [Chain-z: Mastering Asynchronous Execution with RxJava, A Game-Changer for Java-Based Applications](https://medium.com/@manishmishra039/chain-z-mastering-asynchronous-execution-with-rxjava-a-game-changer-for-java-based-applications-3e25cbe57ef8). This blog covers the key benefits of using Chain-z for Asynchronous Execution along with a detailed example of its usage. Chain-Z is a powerful library built on top of RxJava that provides a neat and efficient way of orchestrating asynchronous tasks. It takes care of the boilerplate code needed for async execution. Thus, it provides developers the efficiency to focus on coding the business logic along with the flexibility to execute code asynchronously as per the requirements. 
```

Comment:
looks good can we condense the text of this section.
This blog covers the key benefits of using Chain-z for Asynchronous Execution along with a detailed example of its usage. Chain-Z is a powerful library built on top of RxJava that provides a neat and efficient way of orchestrating asynchronous tasks. It takes care of the boilerplate code needed for async execution. Thus, it provides developers the efficiency to focus on coding the business logic along with the flexibility to execute code asynchronously as per the requirements.

Reply:
@Nishant-sehgal, Done, please review if it looks fine now